src/clims/api/serializers/models/extensible_property.py

```diff
@@ -5,11 +5,5 @@
 
 
 class ExtensiblePropertySerializer(serializers.Serializer):
-    id = serializers.CharField(read_only=True)
-    version = serializers.IntegerField(read_only=True)
     name = serializers.CharField(read_only=True)
-    display_name = serializers.CharField(read_only=True)
-    value = serializers.SerializerMethodField(method_name="get_value_with_correct_type")
-
-    def get_value_with_correct_type(self, obj):
-        return obj.value
+    value = serializers.CharField()
```

Comment:
Why have all the other fields been removed? (Just for my understanding)

Reply:
This was something that I had a specific session with Steinar about, asking him if it was okay. The reason was that the deleted fields most likely aren't going to be used in the UI. The extensible property field only return something if it already has a value. At the start, with a blank page, extensible field is an empty array. This means that fields like display-name are fetched from other static calls. Furthermore, "method field" don't allow writes, which we wants here.